forecasttools/data.py

```diff
@@ -0,0 +1,478 @@
+"""
+Retrieves US 2020 Census data,
+NHSN (as of 2024-09-26) influenza
+hospitalization counts, NHSN
+(as of 2024-09-26) COVID-19
+hospitalization counts, and
+a forecasts for a single
+jurisdiction.
+"""
+
+import os
+
+os.environ["JAX_TRACEBACK_FILTERING"] = "off"
+
+import os
+
+import arviz as az
+import jax.numpy as jnp
+import jax.random as jr
+import matplotlib.pyplot as plt
+import numpy as np
+import numpyro as npro
```

Comment:
```suggestion
import numpyro
```

match our convention in pyrenew